common/src/main/java/dev/lavalink/youtube/clients/skeleton/NonMusicClient.java

```diff
@@ -127,9 +125,14 @@ protected JsonBrowser loadTrackInfoFromInnertube(@NotNull YoutubeAudioSourceMana
             config.withRootField("params", params);
         }
 
-        String payload = config.withPlaybackSignatureTimestamp(signatureCipher.scriptTimestamp)
-            .setAttributes(httpInterface)
-            .toJsonString();
+        String payload = config.setAttributes(httpInterface).toJsonString();;
+        if (this.requiresJSScript()) {
```

Comment:
The existing code style rarely, if at all, uses `this`

```suggestion
        if (requiresJSScript()) {
```